assets/prototype/domain/shared/predicates/datetimes/filters/tests/data.ts

```diff
@@ -7,34 +7,100 @@ const dates: Datetime[] = [
 		capacity: -1,
 		isActive: true,
 		sold: 0,
+		dbId: 2,
+		name: 'Hello',
+		description: 'Test',
+		endDate: 'some',
+		isExpired: false,
+		isPrimary: false,
+		isSoldOut: false,
+		isUpcoming: false,
+		isDeleted: false,
+		status: DatetimeStatus.active,
+		length: null,
+		order: 0,
+		reserved: 0,
+		startDate: 'end',
 	},
 	{
 		id: 'RGF0ZXRpbWU6ODM=',
 		capacity: 10,
 		isActive: true,
 		sold: 5,
 		status: DatetimeStatus.active,
+		dbId: 3,
+		name: 'Hello',
+		description: 'Test',
+		endDate: 'some',
+		isExpired: false,
+		isPrimary: false,
+		isSoldOut: false,
+		isUpcoming: false,
+		isDeleted: false,
+		length: null,
+		order: 0,
+		reserved: 0,
+		startDate: 'end',
 	},
```

Comment:
initially I didn't want to mess up the test data set by @nicolad and then left it as it is. I think we should consider using centralized test data instead of duplicating it everywhere.